<launch>
    <rosparam command="load" file="$(find bb_equipment_tf_publisher)/config/$(env ROBOT_COMPONENT_NODE)_equipment_transforms.yaml"/>
    <node pkg="bb_equipment_tf_publisher" type="bb_equipment_tf_publisher" name="equipment_tf_publisher" output="screen" ns="$(env ROBOT_COMPONENT_NODE)"/>
    <node pkg="bb_equipment_tf_publisher" type="bb_equipment_tf_marker_server" name="equipment_tf_marker_server" output="screen" ns="$(env ROBOT_COMPONENT_NODE)"/>
</launch>
